HeavyIonsAnalysis/JetAnalysis/test/runForestAOD_PbPb_MB_92X.py

```diff
@@ -110,9 +114,11 @@
 process.load("HeavyIonsAnalysis.JetAnalysis.pfcandAnalyzer_cfi")
 process.pfcandAnalyzer.skipCharged = False
 process.pfcandAnalyzer.pfPtMin = 0
+process.pfcandAnalyzer.doVS = cms.untracked.bool(False)
 process.load("HeavyIonsAnalysis.JetAnalysis.pfcandAnalyzerCS_cfi")
 process.pfcandAnalyzerCS.skipCharged = False
 process.pfcandAnalyzerCS.pfPtMin = 0
+process.pfcandAnalyzer.doVS = cms.untracked.bool(False)
```

Comment:
@mverwe - Is this a typo? I think it should be 
process.pfcandAnalyzerCS.doVS = cms.untracked.bool(False)